lib/http3/common.c

```diff
@@ -57,6 +57,51 @@ struct st_h2o_http3_ingress_unistream_t {
 
 const ptls_iovec_t h2o_http3_alpn[1] = {{(void *)H2O_STRLIT("h3-25")}};
 
+static struct {
+    pthread_mutex_t lock;
+    time_t last_reported;
+    size_t failures;
+    size_t successes;
```

Comment:
I think that the type of counters should be `uint64_t`. These counts are unrelated to the amount of memory that can be allocated, and needs to be greater than 32-bit unsigned int even in 32-bit environments.